client/signup/previous-step-button/index.jsx

```diff
@@ -30,27 +28,31 @@ export default React.createClass( {
 		const previousStepName = signupUtils.getPreviousStepName( this.props.flowName, this.props.stepName ),
 			{ stepSectionName } = find( this.props.signupProgress, { stepName: previousStepName } );
 
-		return signupUtils.getStepUrl( this.props.flowName, previousStepName, stepSectionName, i18n.getLocaleSlug() );
-	},
+		return signupUtils.getStepUrl( this.props.flowName, previousStepName, stepSectionName, getLocaleSlug() );
+	}
 
-	recordClick() {
+	recordClick = () => {
 		analytics.tracks.recordEvent( 'calypso_signup_previous_step_button_click', {
```

Comment:
If you're up for it in another PR we can update this component to use redux [analytics middleware](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics). This makes testing much easier.

Reply:
Thanks! Good to know!